srml/system/src/lib.rs

```diff
@@ -283,6 +283,14 @@ impl<T: Trait> Module<T> {
 		storage::unhashed::get(well_known_keys::EXTRINSIC_INDEX)
 	}
 
+	/// Gets a total length of all executed extrinsics.
+	pub fn extrinsics_len() -> u32 {
+		let index = Self::extrinsic_index().unwrap_or_default();
```

Comment:
I'm really hasitant about computing that after every extrinsic, but was unsure  if it's ok to use storage for that. Would be `storage::unhashed` most likely alike `EXTRINSIC_INDEX`.

Thoughts?